### PR TITLE
UPDATE gems

### PR DESCRIPTION
**Pull request by cdmo**
* actioncable 6.0.3.3 → 6.0.3.4
[changelog](https://github.com/rails/rails/blob/v6.0.3.4/actioncable/CHANGELOG.md#rails-6034-october-07-2020)

* actionmailbox 6.0.3.3 → 6.0.3.4
[changelog](https://github.com/rails/rails/blob/v6.0.3.4/actionmailbox/CHANGELOG.md#rails-6034-october-07-2020)

* actionmailer 6.0.3.3 → 6.0.3.4
[changelog](https://github.com/rails/rails/blob/v6.0.3.4/actionmailer/CHANGELOG.md#rails-6034-october-07-2020)

* actionpack 6.0.3.3 → 6.0.3.4
[changelog](https://github.com/rails/rails/blob/v6.0.3.4/actionpack/CHANGELOG.md#rails-6034-october-07-2020)

* actiontext 6.0.3.3 → 6.0.3.4
[changelog](https://github.com/rails/rails/blob/v6.0.3.4/actiontext/CHANGELOG.md#rails-6034-october-07-2020)

* actionview 6.0.3.3 → 6.0.3.4
[changelog](https://github.com/rails/rails/blob/v6.0.3.4/actionview/CHANGELOG.md#rails-6034-october-07-2020)

* activejob 6.0.3.3 → 6.0.3.4
[changelog](https://github.com/rails/rails/blob/v6.0.3.4/activejob/CHANGELOG.md#rails-6034-october-07-2020)

* activemodel 6.0.3.3 → 6.0.3.4
[changelog](https://github.com/rails/rails/blob/v6.0.3.4/activemodel/CHANGELOG.md#rails-6034-october-07-2020)

* activerecord 6.0.3.3 → 6.0.3.4
[changelog](https://github.com/rails/rails/blob/v6.0.3.4/activerecord/CHANGELOG.md#rails-6034-october-07-2020)

* activestorage 6.0.3.3 → 6.0.3.4
[changelog](https://github.com/rails/rails/blob/v6.0.3.4/activestorage/CHANGELOG.md#rails-6034-october-07-2020)

* activesupport 6.0.3.3 → 6.0.3.4
[changelog](https://github.com/rails/rails/blob/v6.0.3.4/activesupport/CHANGELOG.md#rails-6034-october-07-2020)

* better_errors 2.7.1 → 2.8.3
[changelog](https://github.com/BetterErrors/better_errors/blob/master/CHANGELOG.md)

* bootsnap 1.4.6 → 1.4.8
[changelog](https://github.com/Shopify/bootsnap/blob/master/CHANGELOG.md#148)

* capybara 3.32.2 → 3.33.0
[changelog](https://github.com/teamcapybara/capybara/blob/master/History.md#version-3330)

* crack 0.4.3 → 0.4.4
[changelog](https://github.com/jnunemaker/crack/blob/master/History)

* ddtrace 0.40.0 → 0.41.0
[changelog](https://github.com/DataDog/dd-trace-rb/blob/master/CHANGELOG.md#0410---2020-09-30)

* diff-lcs 1.3 → 1.4.4
[changelog](https://github.com/halostatue/diff-lcs/blob/master/History.md#144--2020-07-01)

* dry-configurable 0.11.5 → 0.11.6
[changelog](https://github.com/dry-rb/dry-configurable/blob/master/changelog.yml)

* dry-initializer 3.0.3 → 3.0.4
[changelog](https://github.com/dry-rb/dry-initializer/blob/master/changelog.yml)

* dry-logic 1.0.6 → 1.0.8
[changelog](https://github.com/dry-rb/dry-logic/blob/master/changelog.yml)

* dry-schema 1.5.1 → 1.5.5
[changelog](https://github.com/dry-rb/dry-schema/blob/master/changelog.yml)

* dry-validation 1.5.1 → 1.5.6
[changelog](https://github.com/dry-rb/dry-validation/blob/master/changelog.yml)

* erb_lint 0.0.34 → 0.0.35

* factory_bot 5.2.0 → 6.1.0
[changelog](https://github.com/thoughtbot/factory_bot/blob/master/NEWS.md#610-july-8-2020)

* factory_bot_rails 5.2.0 → 6.1.0
[changelog](https://github.com/thoughtbot/factory_bot_rails/blob/master/NEWS.md#610-july-8-2020)

* json 2.3.0 → 2.3.1
[changelog](https://github.com/flori/json/blob/master/CHANGES.md#2020-06-30-231)

* nio4r 2.5.3 → 2.5.4
[changelog](https://github.com/socketry/nio4r/blob/v2.5.4/CHANGES.md#254-2020-09-16)

* parser 2.7.1.3 → 2.7.2.0
[changelog](https://github.com/whitequark/parser/blob/v2.7.2.0/CHANGELOG.md)

* public_suffix 4.0.5 → 4.0.6
[changelog](https://github.com/weppos/publicsuffix-ruby/blob/v4.0.6/CHANGELOG.md#406)

* rack-protection 2.0.8.1 → 2.1.0

* rails 6.0.3.3 → 6.0.3.4

* rails-controller-testing 1.0.4 → 1.0.5

* railties 6.0.3.3 → 6.0.3.4
[changelog](https://github.com/rails/rails/blob/v6.0.3.4/railties/CHANGELOG.md#rails-6034-october-07-2020)

* redis 4.2.1 → 4.2.2
[changelog](https://github.com/redis/redis-rb/blob/v4.2.2/CHANGELOG.md#422)

* regexp_parser 1.7.1 → 1.8.2
[changelog](https://github.com/ammar/regexp_parser/blob/master/CHANGELOG.md#182---2020-10-11---janosch-müller)

* rspec-core 3.9.2 → 3.9.3
[changelog](https://github.com/rspec/rspec-core/blob/main/Changelog.md)

* rubocop 0.85.1 → 0.93.1
[changelog](https://github.com/rubocop-hq/rubocop/blob/master/CHANGELOG.md#0931-2020-10-12)

* rubocop-ast 0.0.3 → 0.8.0
[changelog](https://github.com/rubocop-hq/rubocop-ast/blob/master/CHANGELOG.md#080-2020-10-12)

* rubocop-performance 1.6.1 → 1.8.1
[changelog](https://github.com/rubocop-hq/rubocop-performance/blob/master/CHANGELOG.md#181-2020-09-19)

* rubocop-rails 2.6.0 → 2.8.1
[changelog](https://github.com/rubocop-hq/rubocop-rails/blob/master/CHANGELOG.md#281-2020-09-16)

* rubocop-rspec 1.40.0 → 1.43.2
[changelog](https://github.com/rubocop-hq/rubocop-rspec/blob/master/CHANGELOG.md#1432-2020-08-25)

* sidekiq 6.1.1 → 6.1.2
[changelog](https://github.com/mperham/sidekiq/blob/master/Changes.md#612)

* sinatra 2.0.8.1 → 2.1.0
[changelog](https://github.com/sinatra/sinatra/blob/master/CHANGELOG.md#210--2020-09-05)

* spring 2.1.0 → 2.1.1
[changelog](https://github.com/rails/spring/blob/master/CHANGELOG.md#211)

* sprockets-rails 3.2.1 → 3.2.2

* view_component 2.18.0 → 2.19.1
[changelog](https://github.com/github/view_component/blob/master/CHANGELOG.md#2191)

* warden 1.2.8 → 1.2.9
[changelog](https://github.com/wardencommunity/warden/blob/master/CHANGELOG.md#version-129--2020-08-31)

* webmock 3.8.3 → 3.9.2
[changelog](https://github.com/bblimke/webmock/blob/v3.9.2/CHANGELOG.md#392)

* webpacker 5.1.1 → 5.2.1
[changelog](https://github.com/rails/webpacker/blob/v5.2.1/CHANGELOG.md#521---2020-08-17)

### CODE REVIEW

**Comment by cdmo** [comment]
regarding changes to `let` limits:

>  Note: let can enhance readability when used sparingly (1,2, or maybe 3 declarations) in any given example group, but that can quickly degrade with overuse. YMMV.

[source](http://rspec.info/documentation/3.9/rspec-core/RSpec/Core/MemoizedHelpers/ClassMethods.html#let-instance_method)

and

> Until you need to break out the big guns like shared examples, avoid DSL constructs like subject, let, its, and before. Stick to your old friends: variables, methods, and classes.

[src](https://thoughtbot.com/blog/lets-not)

Read through this too: https://github.com/rubocop-hq/rubocop-rspec/issues/862

I think `let` can be confusing and should definitely not be used if there are no cases where the variable _is not_ loaded anyway, which we had a bunch of.

File: app/models/call.rb [hunk 1]
```diff
@@ -11,7 +11,7 @@ def initialize(record:)
   end
 
   def item_list
-    fields.dig('itemList')
+    fields['itemList']
```
Comment (banukutlu) [inline app/models/call.rb:14]: no digging 👀

**Review by banukutlu — APPROVED** [review]
LGTM 😎

File: app/models/call.rb (hunk 1 above)
Comment (cdmo) [reply app/models/call.rb:14]: yeah... makes sense, no digging on single item targets.

**Comment by cdmo** [comment]
um, LGTM?

**Pull request merged**